src/reducers/transitionProperties.ts

```diff
@@ -56,6 +54,14 @@ export const transitionPropertiesReducer: Reducer<TransitionPropertiesState> = (
                 ...state,
                 privateTopic: setPrivateTopicAction.privateTopic
             };
+        case SEND:
+            const sendAction = <SendAction>action;
+            send(sendAction.component, sendAction.stateMachine, sendAction.messageType, sendAction.jsonMessageString);
```

Comment:
Is it really necessary to write test in a portion of code that don't change the redux state ?